sdk/storage/inc/datalake/path_client.hpp

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/storage_credential.hpp"
+#include "common/storage_url_builder.hpp"
+#include "datalake/file_system_client.hpp"
+#include "datalake_options.hpp"
+#include "http/pipeline.hpp"
+#include "protocol/datalake_rest_client.hpp"
+
+#include <memory>
+#include <string>
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  struct ReadPathResponse
+  {
+    Azure::Core::Http::BodyStream* Body;
+    std::string AcceptRanges;
+    std::string CacheControl;
+    std::string ContentDisposition;
+    std::string ContentEncoding;
+    std::string ContentLanguage;
+    int64_t ContentLength = int64_t();
+    std::string ContentRange;
+    std::string ContentType;
+    std::string ContentMD5;
+    std::string Date;
+    std::string ETag;
+    std::string LastModified;
+    std::string RequestId;
+    std::string Version;
+    std::string ResourceType;
+    std::string LeaseDuration;
+    std::string LeaseState;
+    std::string LeaseStatus;
+    std::string ContentMd5;
+    std::map<std::string, std::string> Metadata;
+  };
+
+  struct GetPathPropertiesResponse
+  {
+    std::string AcceptRanges;
```

Comment:
what's the point of this member?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Ranges

Reply:
I meant, why do we need to expose this to end users? How do we expect our users to use this?